### ISSUE TITLE
How to block if it doesn't contain keyword

### ISSUE THREAD

**Issue by ddaaii** [maripo/CustomFilter#143]
How do I block elements if it doesn't contain keywords instead of the default 'blocking if it contains'?

**Comment by andreicaba** [comment]
Indeed, how do you do that?... Tried the regex !(.*keyword to be contained.*)    and... it didn't work(!)...
Does the regex work?!...

**Comment by andreicaba** [comment]
Nevermind, I actually did it with Regex. I checked both „Complete Match” and „Regex” and then used something like this:

^((?!keyword).)*$

by the way, for some reason the escaping char can't be added there (this char:   \   )
